BlazorDB.sln

```diff
@@ -1,11 +1,11 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31213.239
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32901.215
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorDB", "src\BlazorDB\BlazorDB.csproj", "{D0180843-D92A-44B7-9893-F2450415F208}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorDB.Example", "example\BlazorDB.Example\BlazorDB.Example.csproj", "{155CD782-6CEF-4115-9A04-1864A2CDFC90}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "BlazorDB.Example", "BlazorDB.Example\BlazorDB.Example.csproj", "{A47C9FCB-70BC-4B16-B5BB-68C8F6B1012E}"
```

Comment:
Just curious why we are moving this out of the `example` folder?

Reply:
This was unintentional. Will be fixed on next commit.